dts/arm/nuvoton/npcx.dtsi

```diff
@@ -101,6 +101,18 @@
 			label = "SCFG";
 		};
 
+		mdc: mdc@4000c000 {
+			compatible = "syscon";
+			reg = <0x4000c000 0xa>;
+			reg-io-width = <1>;
+		};
+
+		mdc_header: mdc@4000c00a {
+			compatible = "syscon";
+			reg = <0x4000c00a 0x4>;
+			reg-io-width = <2>;
+		};
+
```

Comment:
The DTS change should go in a different commit.

Reply:
Done